### PR TITLE
OOP school library: decorate a class

### PR DESCRIPTION
**Pull request by fabgrel10**
In this PR:

- [x] Create a class Nameable.
- [x] Person class now inherits from Nameable.
- [x] Create decorators in base_decorator file.

### CODE REVIEW

**Review by BrianSammit — CHANGES_REQUESTED** [review]
# CHANGES REQUIRED 🔴 🛑
Hello @fabgrel10, Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! 💯 
## Highlights

- Your project is professional 
- Your repo is professional 
- Well structured files 

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

cat


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app/person.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+require_relative './nameable'
+
 class Person
```
[the remark this answers is not in the document]
Comment (fabgrel10) [reply app/person.rb]: My bad, I surely removed the inheritance while I was testing the project. It's fixed now, and the code works as expected.

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @fabgrel10 👋🏻, 

Great job making the requested changes 💪🏻!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

rosycheeks-mochi-peach


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by fabgrel10** [comment]
Thank you for your reviews @Cathella and @BrianSammit!

**Pull request merged**